package Exception;

public class Exception {
    /*exception
    * try catsh finally custom exception
    * debug trong quá trình phát triển dự án package -> java.util
    * debug tức là chương trình đang chạy tự dưng sập ko biết nguyên nhân do đâu --> phải dò lỗi */
}
